api/auth.go

```diff
@@ -103,7 +103,7 @@ func (a *API) maybeLoadUserOrSession(ctx context.Context) (context.Context, erro
 	}
 
 	var session *models.Session
-	if claims.SessionId != "" {
+	if claims.SessionId != "" && claims.SessionId != uuid.Nil.String() {
```

Comment:
I'm OK with this condition, but also make sure that a few lines below there's a `models.IsNotFound(err)` condition. A missing session is not an "error" in these routes and `withSession` should be `nil` IMO.